components/Layout/Layout.js

```diff
@@ -0,0 +1,14 @@
+import Nav from "../Footer/Nav";
+import Header from "../Header/Header";
+
+const Layout = ({ children }) => {
+  return (
+    <div>
```

Comment:
if you don't style the div here, you could use the Fragments <> and </> to avoid unnecessary divs :)